src/bundle/Resources/public/js/alloyeditor/src/widgets/ez-embed-base.js

```diff
@@ -3,6 +3,8 @@ const IS_LINKED_CLASS = 'is-linked';
 const SHOW_EDIT_LINK_TOOLBAR_ATTR = 'data-show-edit-link-toolbar';
 const DATA_ALIGNMENT_ATTR = 'ezalign';
 const ENTER_KEY_CODE = 13;
+const MAX_ACTIVE_CONTENT_LOAD_REQUESTS = 20;
```

Comment:
What if there will be 10 richtext fields and each will have 25 embeds inside? The problem will still occur.
We talked about this briefly with @alongosz and @Nattfarinn and think we should change behavior here to avoid the requests at all.

The information about the embedded contents should be added by the backend into some data-attribute as a list of content objects. The alloyEditor could use that information on init time to show the needed content name and avoid requests

Reply:
@dew326 sorry, but I would assume that `MAX_ACTIVE_CONTENT_LOAD_REQUESTS` will be for all the rich text fields, but not per each individually. And it might be a good temporary solution. Because getting that info from the back-end is more complex change. Taking into account that a similar approach is used in v3, but GraphQL is used instead of REST.

Reply:
@SerheyDolgushev We're talking about changing the approach in general, so we don't have to do X number of request at all. It still may happen, no matter what number you'll put in `MAX_ACTIVE_CONTENT_LOAD_REQUESTS`, that you'll exceed server limits (because of other request done to server by other fieldtypes or features) so it is better to avoid it in a first place.

Reply:
>Taking into account that a similar approach is used in v3, but GraphQL is used instead of REST.

Can you show me where it is done?

Reply:
I mean v3 uses GraphQL instead of the REST, but it also fetches info using separate request per each embed content/image.

Reply:
@bdunogier please correct me if I`m wrong.

Reply:
I'm gonna have to correct you unfortunately @SerheyDolgushev : GraphQL is only used by the sub-items list.